rusk-wallet/src/lib/store.rs

```diff
@@ -50,15 +56,29 @@ impl LocalStore {
         }
 
         // attempt to load and decode wallet
-        let bytes = fs::read(&path)?;
-        let bytes = decrypt(&bytes, pwd)?;
+        let mut version = 0u8;
+        let mut bytes = fs::read(&path)?;
 
-        if bytes.len() != SEED_SIZE {
-            return Err(Error::WalletFileCorrupted);
+        // check for version information (strips it if present)
+        if bytes[1] == 0 && bytes[2] == 0 {
+            version = bytes[0];
+            bytes.drain(0..3);
         }
 
+        // decrypt and interpret file contents
         let mut seed = [0u8; SEED_SIZE];
-        seed.copy_from_slice(&bytes);
+        match version {
+            0 | 1 => {
+                bytes = decrypt(&bytes, pwd)?;
+                if bytes.len() != SEED_SIZE {
+                    return Err(Error::WalletFileCorrupted);
+                }
+                seed.copy_from_slice(&bytes);
+            }
+            _ => {
+                return Err(Error::UnknownFileVersion);
+            }
+        };
```

Comment:
I am aware this part could be slightly cleaner but right now we really don't have any differences between versions, so I'm treating no version (`0`) and v1 (`1`) as the same thing. Once we start adding features this will naturally evolve to accommodate the new stuff.